python/samples/agentchat_graphrag/requirements.txt

```diff
@@ -1,3 +1,4 @@
 autogen-agentchat
 autogen-ext
-pyyaml
\ No newline at end of file
+pyyaml
+graphrag>=2.2.1
```

Comment:
This isn't needed since it in in the pyproject.toml right?

Reply:
Maybe we should install the ext dep with [graphrag] extra enabled to keep things in sync?